zetaclient/testdata/evm/chain_1_block_ethrpc_trimmed_19330473.json

```diff
@@ -0,0 +1,2049 @@
+{
```

Comment:
Isn't it some way to trim these data a bit?

Reply:
It was a trimmed version. We typically need one archived block for inbound gas token test and one for outbound. Text compression is probably the way to go if we do need many many archived blocks.